WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -328,10 +328,12 @@ class WordPressAppDelegate: UIResponder, UIApplicationDelegate {
 
         setupWordPressExtensions()
 
-        // Start proactively exporting WP data in the background if the conditions are fulfilled.
-        // This needs to be called after `setupWordPressExtensions` because it updates the stored data.
-        DispatchQueue.global().async {
-            ContentMigrationCoordinator.shared.startOnceIfNeeded()
+        if FeatureFlag.contentMigration.enabled {
+            // Start proactively exporting WP data in the background if the conditions are fulfilled.
+            // This needs to be called after `setupWordPressExtensions` because it updates the stored data.
+            DispatchQueue.global().async {
+                ContentMigrationCoordinator.shared.startOnceIfNeeded()
```

Comment:
We could also consider a `guard` inside this call instead, but this approach felt like it separated concerns better to me.

Reply:
> We could also consider a guard inside startOnceIfNeeded() instead

To add some context, this is already done here:

https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/WordPress/Classes/Utility/Migration/ContentMigrationCoordinator.swift#L115-L117

But since the tracking code was placed in the `else` part of the guard, it gets called every time. Adding another check higher up the chain would help prevent this from firing. 👍🏼